packages/app/src/cli/models/extensions/functions.ts

```diff
@@ -143,9 +144,8 @@ export function createFunctionSpec<TConfiguration extends FunctionConfigType = F
     configSchema: BaseFunctionConfigurationSchema,
     gated: false,
     registrationLimit: spec.registrationLimit ?? blocks.functions.defaultRegistrationLimit,
-
     category: (): ExtensionCategory => 'function',
   }
 
-  return {...defaults, ...spec}
+  return {...defaults, ...spec, surface: 'none'}
```

Comment:
why is this needed? functions won't ever need surface, is this because of some typings issue?

Reply:
Yeah, good catch.  

it looks like I added this to makes types pass, but we've since made changes that mean this is not needed.  So i've removed the concept of surface from this file entirely.